### ISSUE TITLE
What we will see on the server side?

### ISSUE THREAD

**Issue by NoamRotter** [GoogleChrome/ip-protection#44]
Hi,
For the traffic that will go through the proxy servers,
Do we expect to see any other changes/additions in the HTTP request/response headers?
Or, all the headers will stay intact but only the IP will be of the proxy?

Thanks.

**Comment by DavidSchinazi** [comment]
That's right, the IP will change but we do not expect any changes to HTTP headers.

**Comment by emergencynap** [comment]
Will the TLS fingerprint change?  Reason I ask is because many bot prevention sites use JA3 fingerprinting based on TLS to detect if a user is a bot and present a captcha.

**Comment by DavidSchinazi** [comment]
TLS is still performed end-to-end between Chrome and the website, and IP Protection won't impact TLS. In particular, IP Protection won't change the TLS client hello, and therefore not the JA3 fingerprint either.

**Comment by NoamRotter** [comment]
Thank you!
